lib/interface_quda.cpp

```diff
@@ -3171,26 +3180,30 @@ void callMultiSrcQuda(void **_hp_x, void **_hp_b, QudaInvertParam *param, // col
       }
     }
 
-    quda::GaugeField *collected_gauge = nullptr;
-    quda::GaugeField *collected_milc_fatlink_field = nullptr;
-    quda::GaugeField *collected_milc_longlink_field = nullptr;
+    quda::GaugeField collected_gauge;
+    quda::GaugeField collected_milc_fatlink_field;
+    quda::GaugeField collected_milc_longlink_field;
```

Comment:
Have we tested split grid with this branch?

Reply:
I've tested it locally for staggered, and I believe the CSCS coverage will hit both wilson-type and staggered-type, since there are split grid unit tests. @maddyscientist am I understanding this correctly?

Reply:
I confirmed independently that a few cursory domain wall solves also work!